### PR TITLE
[FT:] Configure periodic reports

### PR DESCRIPTION
**Pull request by miritih**
**Problem**
Currently, the dashboard only allows subscribing to monthly report. 

**Solution**
- Changed  reports sending to be user configurable to either be daily, weekly or monthly. 
- Added weekly and daily Radio boxes on the UI
- Add test to check can send daily, weeky, and monthly reports.  

**Screenshots**

screen shot 2018-07-12 at 11 48 05

**Relevant Pivotal Tracker story**
https://www.pivotaltracker.com/story/show/158532875

### CODE REVIEW

**Review by thoth-ky — APPROVED** [review]
Good work

**Review by simiyu1 — APPROVED** [review]
Good coding looks clean to me...

**Review by loicemeyo — APPROVED** [review]
This is amazing implementation Eric

File: .env.example [hunk 1]
```diff
@@ -7,4 +7,5 @@ export DB_PORT="5432"
 export EMAIL_HOST="smtp.gmail.com"
 export DJMAIL_EMAIL="your email"
 export DJMAIL_PASSWORD="your email password"
-export EMAIL_PORT="587"
\ No newline at end of file
+export EMAIL_PORT="587"
+export SITE_ROOT="http://localhost:8000"
```
Comment (karanjaE) [inline .env.example]: missing single empty line at the end of the file

File: hc/accounts/forms.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class EmailPasswordForm(forms.Form):
 
 
 class ReportSettingsForm(forms.Form):
-    reports_allowed = forms.BooleanField(required=False)
+    reports_allowed = forms.CharField(required=True)
```
Comment (karanjaE) [inline hc/accounts/forms.py:17]: why was the field changed to charfield?

File: hc/accounts/tests/test_profile.py [hunk 3]
```diff
@@ -40,11 +43,14 @@ def test_it_adds_team_member(self):
         for member in self.alice.profile.member_set.all():
             member_emails.add(member.user.email)
 
-        ### Assert the existence of the member emails
-
+        # Assert the existence of the member emails
+
```
Comment (karanjaE) [inline hc/accounts/tests/test_profile.py]: remove the empty spaces

**Review by karanjaE — CHANGES_REQUESTED** [review]

File: hc/accounts/forms.py (hunk 2 above)
Comment (miritih) [reply hc/accounts/forms.py:17]: To accommodate checking daily, weekly or monthly periods,  I'm saving the period as a string. The earlier was a boolean checking if monthly reports were enabled or not. Considering we are checking more than 3 variables here I decided to change this to a charfield.

Comment (karanjaE) [reply hc/accounts/forms.py:17]: got it.

**Review by karanjaE — APPROVED** [review]

**Pull request merged**